newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/FormatExpressionCall.js

```diff
@@ -29,19 +47,19 @@ export const getVisibleParameterTypes = (
   );
 
   if (expressionMetadata.scope.objectMetadata) {
-    return filterOutCodeOnlyParameters(
+    return sliceVisibleParameters(
```

Comment:
For `getVisibleParameterTypes`, I think we could have kept `filterOutCodeOnlyParameters` - though you can indeed also use `sliceVisibleParameters` (maybe to be renamed to `filterVisibleParameters`, after reworking it according to my comment ;)).

The idea is that getVisibleParameterTypes should remove any "code only" parameter, while `filterVisibleParameters` should do the same AND remove any trailing optional parameter that is not filled.